tools/nodejs_api/src_nodejs/connection.js

```diff
@@ -29,16 +56,18 @@ class Connection {
         const queryResult = new QueryResult(this, nodeQueryResult);
         if ('callback' in opts) {
             const callback = opts['callback'];
-            if (typeof callback !== "function" || callback.length !== 2){
+            if (typeof callback !== "function" || callback.length !== 2) {
```

Comment:
Similar here. There is absolutely no need to check if the callback takes 2 params. The user may not be interested in error or result at all.